src/UserActivity.js

```diff
@@ -108,18 +112,19 @@ export default class UserActivity {
      * Start listening to user activity
      */
     start() {
-        this._document.onmousedown = this._onUserActivity;
-        this._document.onmousemove = this._onUserActivity;
-        this._document.onkeydown = this._onUserActivity;
+        this._document.addEventListener('mousedown', this._onUserActivity);
+        this._document.addEventListener('mousemove', this._onUserActivity);
+        this._document.addEventListener('keydown', this._onUserActivity);
```

Comment:
Thanks for doing this! 😁 We should also change the ones in `stop` to call `removeEventListener` now.